pkgs/tools/admin/proxmove/derivation.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
```

Comment:
Why not put this code on default.nix?

Reply:
Yea I can never figure out which is preferable. I've submitted like that before, but then come across packages which do have the split. Personally I agree on the join, so I'll push that update as well.